src/ostree/ot-admin-builtin-status.c

```diff
@@ -204,6 +207,11 @@ ot_admin_builtin_status (int argc, char **argv, OstreeCommandInvocation *invocat
     {
       g_print ("No deployments.\n");
     }
+  else if (opt_query_booted)
```

Comment:
I think this option should come at the very first thing; otherwise in the degenerate case of "failed to find any deployments" the caller would see an unexpected "No deployments" string.  Because we expect callers to parse this we should be strict about what we output.

Also...I would say it becomes an *error* when using this option if we don't find any deployments, right?